.pre-commit-config.yaml

```diff
@@ -56,3 +55,9 @@ repos:
     - id: no-commit-to-branch
     - id: trailing-whitespace
       exclude: ^vendor
+
+- repo: https://github.com/openstack-dev/bashate.git
```

Comment:
Should this just be: `https://github.com/openstack/bashate.git` ( `s/openstack-dev/openstack/` )

Reply:
Fixed.